llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3778,9 +3779,90 @@ template <> Value *SPIRVToLLVM::transValueWithOpcode<OpGroupFMaxNonUniformAMD>(S
 template <> Value *SPIRVToLLVM::transValueWithOpcode<OpExtInst>(SPIRVValue *const spvValue) {
   SPIRVExtInst *const spvExtInst = static_cast<SPIRVExtInst *>(spvValue);
 
-  // Just ignore this set of extended instructions
-  if (m_bm->getBuiltinSet(spvExtInst->getExtSetId()) == SPIRVEIS_NonSemanticInfo)
-    return nullptr;
+  if (m_bm->getBuiltinSet(spvExtInst->getExtSetId()) == SPIRVEIS_NonSemanticInfo) {
+    switch (spvExtInst->getExtOp()) {
+    case 1: { // DebugPrintf
```

Comment:
I think we ought to import the file to LLPC like what we have done for GLSL.std.450.h. Of course, this extended instruction set has just one instruction currently but maybe it will be enlarged in the future.
https://github.com/KhronosGroup/SPIRV-Headers/blob/master/include/spirv/unified1/NonSemanticDebugPrintf.h